### PR TITLE
Recover schedules

### PR DESCRIPTION
**Pull request by richarddubay**

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://ad3babdf-e2b6-4069-ade7-6cc47c2c37da.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/recover-schedules/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by richarddubay** [comment]
recover
remind
transfer
transfer 2

**Comment by richarddubay** [comment]
@jbaxleyiii @johnthepink This is feeling pretty good to me. What am I missing?

File: imports/blocks/recover-schedules/Remind.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 
-const Remind = ({ onSubmit, back }) => {
+type IRemind = {
+  onSubmit: Function,
+  back: Function,
+};
+
+/* eslint-disable max-len */
+const Remind = ({ onSubmit, back }: IRemind) => {
   const reminderFrequency = [
```
Comment (jbaxleyiii) [inline imports/blocks/recover-schedules/Remind.js]: @richarddubay we should move this out of the render function since its a static

Comment (richarddubay) [inline imports/blocks/recover-schedules/Remind.js]: @jbaxleyiii Fixed!

**Pull request merged**